Sources/Unrar/Entry.swift

```diff
@@ -15,7 +15,9 @@ public struct Entry: Equatable {
 
     init(_ header: RARHeaderDataEx) {
         var _header: RARHeaderDataEx = header
-        self.fileName = String(cString: &_header.FileName.0)
+        self.fileName = withUnsafePointer(to: &_header.FileName.0) { ptr -> String in
+            return String(cString: UnsafeRawPointer(ptr).assumingMemoryBound(to: CChar.self))
+        }
```

Comment:
```suggestion
        self.fileName = withUnsafePointer(to: &_header.FileName.0) { String(cString: $0) }
```
I have no idea why you use assumingMemoryBound? I can't find it.

Reply:
It's just typing the pointer to CChar, not strictly required.

Reply:
It's just typing the pointer to CChar, not strictly required.